local-dev.yml

```diff
@@ -55,12 +58,6 @@ services:
       - "6432:6432"
     depends_on:
       - postgis
-  rabbitmq:
```

Comment:
https://github.com/hydroshare/hydroshare/blob/redpanda-events/README.md?plain=1#L119
☝️ needs an update

Reply:
also think we can remove this file
https://github.com/hydroshare/hydroshare/blob/redpanda-events/scripts/templates/docker-compose.template

Reply:
done